tests/gcp/installer/util/test_data/merge_from_forseti_conf.yaml

```diff
@@ -0,0 +1,172 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
```

Comment:
As we discussed, it would be awesome to add test cases for each of the released `forseti_conf.yaml` versions, just to be 100% sure this can be migrated successfully.